testing/chain.go

```diff
@@ -589,3 +589,10 @@ func (chain *TestChain) GetChannelCapability(portID, channelID string) *capabili
 func (chain *TestChain) GetTimeoutHeight() clienttypes.Height {
 	return clienttypes.NewHeight(clienttypes.ParseChainID(chain.ChainID), uint64(chain.GetContext().BlockHeight())+100)
 }
+
+// DeleteKey deletes the specified key from the ibc store.
+func (chain *TestChain) DeleteKey(key []byte) {
```

Comment:
Don't really feel this helper function needs to exist.

Either do this inside the one test case you need it. Or explicitly do the set error receipt in the other test cases,

Reply:
Sorry missed this, can remove the helper in a follow up if you don't think it's worth having 👍